_posts/memory-creation-overflow-bug.md

```diff
@@ -0,0 +1,87 @@
+---
+layout: post
+title:  "Solidity Memory Array Creation Overflow Bug"
+date:   2020-04-02
+author: Solidity Team
+---
+
+# Solidity Memory Array Creation Overflow Bug Announcement
+
+On the 28th of March a bug in the Solidity code generator was reported through the
+<a href="https://bounty.ethereum.org/" target="_blank">Ethereum Foundation Bounty program</a>,
+by John Toman of Certora. The bug is fixed with version 0.6.5, released on [[[Insert Release Date]]].
+
```

Comment:
We assigned a severity level of "low" because we found the bug very hard to exploit in real-world smart contracts.

Reply:
... because we found the bug to be uncommon and at the same time very hard to exploit